mako.5.scd

```diff
@@ -0,0 +1,118 @@
+mako(5)
+
+# NAME
+
+mako - configuration file
+
+# DESCRIPTION
+
+The mako configuration file is located at *~/.config/mako/config* or at
+*$XDG\_CONFIG\_HOME/mako/config*. Each line of the form:
+
+	key=value
+
+Is equivalent to passing *--key=value* to mako from the command line. See
+*mako*(1) for a list of command line options. Note that any quotes used within
```

Comment:
This bouncing back and forth is why I'm slightly hesitant to break these up. Would it make more sense to just alter the flow of the manpage so that users learn they can use the config file before seeing all of the flags? That section on its own is very short and could be right under the help flag section.